### DISCUSSION TITLE
Iframe youtube always continues the video from the time it was last viewed

### DISCUSSION THREAD

**Discussion by DanDiaborn** [community/community#130607] [Programming Help]
### Body

```
 <iframe id="youtube-video" width="560" height="315"
    src="https://www.youtube.com/embed/Y-P9tNkTLXw?si=1VXDflMvbTApKwhg&amp;start=300" title="YouTube video player"
    frameborder="0"
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
    referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
When loading for a second, playback starts from 300 seconds and then rewinds back to the moment at which you finished watching the last time.
How can I make it play from a certain second?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by HemalathaT2** [comment]
Hi Dan,
<iframe id="youtube-video" width="560" height="315"
    src="https://www.youtube.com/embed/Y-P9tNkTLXw?si=1VXDflMvbTApKwhg&start=300&autoplay=1"
    title="YouTube video player"
    frameborder="0"
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
    referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

In this example, the video will start playing from 300 seconds (5 minutes) and will not rewind back to the previous playback position.

Have a good day!

**Comment by niteesh18433** [comment]
helpful #

**Comment by harlangilbran** [comment]
test
